src/arrayMethodSort.js

```diff
@@ -4,9 +4,22 @@
  * Implement method Sort
  */
 function applyCustomSort() {
-  [].__proto__.sort2 = function(compareFunction) {
-    // write code here
-  };
+  // write code here
+  [].__proto__.sort2
+    = function(compareFunction = (a, b) => String(a) > String(b)) {
+      for (let i = 0; i < this.length; i++) {
+        for (let k = i + 1; k < this.length; k++) {
+          if (compareFunction(this[i], this[k]) > 0) {
+            const compare = this[i];
+
+            this[i] = this[k];
+            this[k] = compare;
+          }
+        }
+      }
```

Comment:
let's use [destructuring assignment](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/Destructuring_assignment) for swapping values.

![image](https://github.com/mate-academy/js_array-method-sort/assets/108538921/46840e1b-1d77-4b0f-8fc0-0945b8132003)